code/modules/mob/living/carbon/xenomorph/castes/boiler/abilities_boiler.dm

```diff
@@ -147,27 +159,34 @@ GLOBAL_LIST_INIT(boiler_glob_image_list, list(
 	var/mob/living/carbon/xenomorph/boiler/X = owner
 
 	if(X.xeno_flags & XENO_ZOOMED)
-		to_chat(X, span_notice("We can not prepare globules as we are now. We must stop concentrating into the distance!"))
+		X.balloon_alert(X,"Can't prepare globules while zoomed in!")
 		return
 
 	var/current_ammo = X.corrosive_ammo + X.neuro_ammo
 	if(current_ammo >= X.xeno_caste.max_ammo)
-		to_chat(X, span_notice("We can carry no more globules."))
+		X.balloon_alert(X,"Globule storage full!")
 		return
 
 	succeed_activate()
 	if(istype(X.ammo, /datum/ammo/xeno/boiler_gas/corrosive))
 		X.corrosive_ammo++
-		to_chat(X, span_notice("We prepare a corrosive acid globule."))
+		X.balloon_alert(X,"Acid globule prepared - [X.corrosive_ammo] in reserve.")
 	else
 		X.neuro_ammo++
-		to_chat(X, span_notice("We prepare a neurotoxic gas globule."))
+		X.balloon_alert(X,"Neuro globule prepared - [X.neuro_ammo] in reserve.")
```

Comment:
As a rule try keep balloon alerts very short. They have key information from A: they presumably know what buttons they're pressing, and B: The action button has more info that they can read.